prow/config.yaml

```diff
@@ -51,6 +51,98 @@ presubmits:
     context: pull-kops-e2e-kubernetes-aws
     rerun_command: "/test pull-kops-e2e-kubernetes-aws"
     trigger: "(?m)^/test( all| pull-kops-e2e-kubernetes-aws),?(\\s+|$)"
+  kubernetes-incubator/cri-containerd:
+  - name: pull-cri-containerd-build
+    agent: kubernetes
+    always_run: true
+    context: pull-cri-containerd-build
+    rerun_command: "/test pull-cri-containerd-build"
+    trigger: "(?m)^/test( all| pull-cri-containerd-build),?(\\s+|$)"
+    branches:
+    - master
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:0.4
+        args:
+        - "--repo=github.com/$(REPO_NAME)=$(PULL_REFS)"
```

Comment:
probably need to be `github.com/kubernetes-incubator/$(REPO_NAME)`, as $(REPO_NAME) is `cri-containerd`

```
I1012 05:49:26.041] Call:  git fetch --quiet --tags https://github.com/cri-containerd master +refs/pull/338/head:refs/pr/338
W1012 05:49:26.286] remote: Not Found
W1012 05:49:26.286] fatal: repository 'https://github.com/cri-containerd/' not found
```